openmetadata-service/src/main/java/org/openmetadata/service/events/EventFilter.java

```diff
@@ -30,6 +30,7 @@
 @Provider
 public class EventFilter implements ContainerResponseFilter {
   private static final List<String> AUDITABLE_METHODS = Arrays.asList("POST", "PUT", "PATCH", "DELETE");
+  private static List<String> EXCLUDED_ENDPOINTS = getEndpointsFilters();
```

Comment:
nit: we are initializing static fields like this in other parts of the code.
```suggestion
  private static List<String> EXCLUDED_ENDPOINTS = new ArrayList<>(JwtFilter.EXCLUDED_ENDPOINTS);
  static {
    EXCLUDED_ENDPOINTS.add("v1/analytics/webAnalyticEvent/collect");
  }
```